### PR TITLE
remove consul lock relevant code

### PR DESCRIPTION
**Pull request by qibobo**
[#163317022]

### CODE REVIEW

File: src/autoscaler/operator/config/config.go [hunk 1]
```diff
@@ -93,7 +87,6 @@ type Config struct {
 	ScalingEngine     ScalingEngineConfig           `yaml:"scaling_engine"`
 	Scheduler         SchedulerConfig               `yaml:"scheduler"`
 	AppSyncer         AppSyncerConfig               `yaml:"app_syncer"`
-	Lock              LockConfig                    `yaml:"lock"`
 	DBLock            DBLockConfig                  `yaml:"db_lock"`
 	EnableDBLock      bool                          `yaml:"enable_db_lock"`
```
Comment (boyang9527) [inline src/autoscaler/operator/config/config.go]: if we only support dbLock, then we need to remove this config too.  There will be corresponding changes in main.go.

Comment (qibobo) [reply src/autoscaler/operator/config/config.go]: @boyang9527  You mean while consul lock is removed, so dbLock is mandatory for operator?

Comment (boyang9527) [reply src/autoscaler/operator/config/config.go]: yes

**Comment by qibobo** [comment]
@boyang9527 updated.

**Comment by boyang9527** [comment]
LGTM

**Pull request merged**